connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2079,609 +2063,6 @@ class SparkConnectPlanner(val session: SparkSession) {
     }
   }
 
-  def process(
```

Comment:
I think we can not simply remove this method, since `SparkConnectPlanner#process` is exposed to the extensions